src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -57,19 +57,11 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var installerService = GetPackageInstallerService(document);
             var packageSources = searchNuGetPackages && symbolSearchService != null && installerService?.IsEnabled(document.Project.Id) == true
-                ? await installerService.TryGetPackageSourcesAsync(allowSwitchToMainThread: false, context.CancellationToken).ConfigureAwait(false)
+                ? installerService.TryGetPackageSources()
                 : ImmutableArray<PackageSource>.Empty;
 
-            if (packageSources is null)
-            {
-                // Information about package sources is not available. This code fix cannot provide results for NuGet
-                // packages at this time, but future invocations of the code fix will work. For the current code fix
-                // operation, just treat the package sources as empty.
-                packageSources = ImmutableArray<PackageSource>.Empty;
-            }
```

Comment:
all clients always treated null as empty.  so i've just simplified the contract to: you get the values back if we can, or empty if we cannot.